### ISSUE TITLE
Update stylelint dependency

### ISSUE THREAD

**Issue by japboy** [pascalduez/stylelint-config-css-modules#8]
I've updated `stylelint` from `13.8.0` to `13.12.0` and it causes false positive errors with `stylelint-config-css-modules@2.2.0` below:

```
path/to/file.vue
 119:1  ✖  Unknown rule declaration-block-no-duplicate-custom-properties  declaration-block-no-duplicate-custom-properties
 119:1  ✖  Unknown rule named-grid-areas-no-invalid                       named-grid-areas-no-invalid 
```

and actual file doesnt seem to have problems:

![image](https://user-images.githubusercontent.com/84360/112316287-4ef95400-8cee-11eb-9660-253e20b3a238.png)

These errors are all gone if I remove `stylelint-config-css-modules`

`.stylelintrc.js`

```js
module.exports = {
  extends: [
    'stylelint-config-standard',
    // everything is fine if comment out below
    'stylelint-config-css-modules',
  ],
}
```

**Comment by japboy** [comment]
this doesnt seemt to be relevant to stylelint-config-css-modules actually. closed.
